system/Database/SQLite3/Connection.php

```diff
@@ -207,6 +207,7 @@ protected function _escapeString(string $str): string
 	protected function _listTables(bool $prefixLimit = false): string
 	{
 		return 'SELECT "NAME" FROM "SQLITE_MASTER" WHERE "TYPE" = \'table\''
+			   . ' AND "NAME" NOT LIKE \'sqlite_%\''
```

Comment:
The current query will match not only `sqlite_` prefix but every name like: `sqlite123`, `sqlitewhatever` etc.

It should be:
```suggestion
			   . ' AND "NAME" NOT LIKE \'sqlite!_%\' ESCAPE \'!\'' 
```

Reply:
Glad you're here checking up... will fix.